integrationtests/tests/fixtures/mod.rs

```diff
@@ -141,23 +140,8 @@ pub async fn fixtures(num_peers: u16) -> anyhow::Result<Fixtures> {
 
     // in case we need to output logs using 'cargo test -- --nocapture'
     if base_port == BASE_PORT_INIT {
-        if env::var_os("FEDIMINT_TRACE_CHROME").map_or(false, |x| !x.is_empty()) {
-            let (cr_layer, gaurd) = tracing_chrome::ChromeLayerBuilder::new()
-                .include_args(true)
-                .build();
-            // drop gaurd cause file to written and closed
-            // in this case file will closed after exit of program
-            std::mem::forget(gaurd);
-
-            tracing_subscriber::registry().with(cr_layer).init();
-        } else {
-            tracing_subscriber::fmt()
-                .with_env_filter(
-                    EnvFilter::try_from_default_env()
-                        .unwrap_or_else(|_| EnvFilter::new("info,fedimint::consensus=warn")),
-                )
-                .init();
-        }
+        let chrome = env::var_os("FEDIMINT_TRACE_CHROME").map_or(false, |x| !x.is_empty());
```

Comment:
Kinda tangential, but should we standardize our environment variables on `FM_` prefix?